api_server/sense/test_request_api.py

```diff
@@ -0,0 +1,42 @@
+import requests
+
+def test_endpoints():
+    URL = 'https://sensemodel.herokuapp.com/'
+    PARAMS = {""}
+    r = requests.get(URL)
```

Comment:
I think it could be good to have a more explicit variable name like "response" or something like that.